<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataMiningWindow</class>
 <widget class="QWidget" name="DataMiningWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>710</height>
    </rect>
   </property>
   <widget class="QLabel" name="table_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Название таблицы
со свойствами</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="properties_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Названия свойств</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="table_name_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>objects</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="properties_text">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>370</width>
      <height>90</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;x1&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;x2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="class_table_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Название таблицы
с классами</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="class_table_name_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>classes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_db_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть базу данных</string>
    </property>
   </widget>
   <widget class="QLabel" name="database_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файлы
с базой данных</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="database_name_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>test.sqlite</string>
    </property>
   </widget>
   <widget class="QLabel" name="log_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Лог программы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_value">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>530</width>
      <height>120</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="parse_names_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Прочитать названия</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_column_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Название столбца
с id</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="time_column_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Название столбца
с тактами</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_column_name_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time_column_name_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="class_column_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Название столбца
с классами</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="class_column_name_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>class</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_tree_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество
деревьев</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_tree_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_tree_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить деревья</string>
    </property>
   </widget>
   <widget class="QLabel" name="serialize_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла с
сериализацией</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="serialize_name_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>serialized</string>
    </property>
   </widget>
   <widget class="QLabel" name="rules_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла с
описанием на ЯПЗ</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="rules_name_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>rules</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_trees_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>490</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить деревья</string>
    </property>
   </widget>
   <widget class="QLabel" name="inner_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя файла с
 дампом ПЗ</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inner_name_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>370</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>inner</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Главное окно</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>8</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
